superdesk/commands/delete_text_doc.py

```diff
@@ -0,0 +1,58 @@
+from eve_elastic import get_es
+import elasticsearch
+import superdesk
+
+class DeleteDocCommand(superdesk.Command):
+    '''
+
+    Delete a Text Archive document from both Mongodb and ElasticSearch by supplying a GUID
+
+    '''
+    option_list = [
+        superdesk.Option('--guids', '-i', dest='guids', required=True,
+                         help='pass guids for articles to delete')
+    ]
+
+    def run(self, guids):
+        guids = guids.split(",")
+
+        elastic_locations = [('superdesk', 'archive')]
+        bulk_commands = []
+
+        for guid in guids:
+            for (_index, _type) in elastic_locations: 
+                data = {
+                    '_op_type': 'delete',
+                    '_index': _index,
+                    '_type': _type,
+                    '_id': guid,
+                }
+
+                bulk_commands.append(data)
+
+        try:
+            es = get_es(superdesk.app.config['ELASTICSEARCH_URL'])
+            elasticsearch.helpers.bulk(es, bulk_commands)
+        except elasticsearch.helpers.BulkIndexError as BulkIndexError:
+            print(BulkIndexError)
+        except elasticsearch.exceptions.NotFoundError as nfe:
+            print(nfe)
+
+        try:
+            superdesk.get_resource_service('archive').delete(lookup={'_id': {'$in': guids}})
```

Comment:
The ticket asks for a delete from Text Archive so the collection should be `archived`. It doesn't say the deletion from the `archive` which is the production.